lib/chirp-workflow/core/src/ctx/api.rs

```diff
@@ -128,6 +154,25 @@ impl ApiCtx {
 		.await?
 	}
 
+	/// Dispatch a new workflow with tags and wait for it to complete. Has a 60s timeout.
+	pub async fn tagged_workflow<I>(
+		&self,
+		tags: &serde_json::Value,
+		input: I,
+	) -> GlobalResult<<<I as WorkflowInput>::Workflow as Workflow>::Output>
+	where
+		I: WorkflowInput,
+		<I as WorkflowInput>::Workflow: Workflow<Input = I>,
+	{
+		let workflow_id = self.dispatch_tagged_workflow(tags, input).await?;
+
+		tokio::time::timeout(
```

Comment:
Is each workflow run dispatched from an async handler? So blocking like this in a workflow is fine? The mental model (or assumption rather) that I had was that if a workflow needed to do something a certain amount of time later, it would put something in the db that would be woken up at that point.

Reply:
This is the api context, used in the api layer. this is technically outside of the workflow ecosystem